### PR TITLE
feat: add redirects and update path for arbitrum faq

### PR DESCRIPTION
**Pull request by saihaj**
add redirects and update path for arbitrum faq

### CODE REVIEW

**Comment by benface** [comment]
Hey @saihaj, looks like we did some conflicting work. 😬 I worked directly in [Michael's branch, which was merged](https://github.com/graphprotocol/docs/pull/415). Sorry about that!

I went with a shorter URL segment (`arbitrum` instead of `scaling-with-arbitrum`), but I didn't add a redirect like you did. Maybe we can repurpose this PR to just add the redirect?

File: nginx.conf [hunk 1]
```diff
@@ -84,6 +84,7 @@ http {
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/supported-networks/cosmos/$                   $scheme://$http_host/docs/$1/cookbook/cosmos/                               permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/supported-networks/near/$                     $scheme://$http_host/docs/$1/cookbook/near/                                 permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/developing/defining-a-subgraph/$              $scheme://$http_host/docs/$1/developing/creating-a-subgraph/                permanent;
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/arbitrum-faq/$                                $scheme://$http_host/docs/$1/arbitrum/faq/                                  permanent;
```
Comment (benface) [inline nginx.conf:87]:
```suggestion
        rewrite ^/docs/([a-zA-Z][a-zA-Z])/arbitrum-faq/$                                $scheme://$http_host/docs/$1/arbitrum/arbitrum-faq/                                  permanent;
```

File: website/next.config.js [hunk 2]
```diff
@@ -52,6 +52,11 @@ export default withNextra({
       destination: '/en/substreams/README/',
       permanent: false,
     },
+    {
+      source: '/en/arbitrum-faq/',
+      destination: '/en/arbitrum/faq/',
+      permanent: true,
+    },
```
Comment (benface) [inline website/next.config.js:59]:
```suggestion
    {
      source: '/en/arbitrum-faq/',
      destination: '/en/arbitrum/arbitrum-faq/',
      permanent: true,
    },
```

Comment (saihaj) [reply website/next.config.js:59]: I did change the path cause just to make it more easier. But can revert it
 
CleanShot 2023-06-15 at 12 16 31

Comment (benface) [reply website/next.config.js]: Oh sorry, I totally missed that change. I see that you reverted it. Sorry about that, you're right that it's a bit repetitive as it is... but oh well.

**Review by benface — APPROVED** [review]

**Pull request merged**